doc/source/notebooks/dataset_gsm8k_sft.ipynb

```diff
@@ -34,7 +34,7 @@
     "    load_text_tokenizer,\n",
     "    setup_gangs,\n",
     ")\n",
-    "from fairseq2.recipes.config import GangSection\n",
+    "from fairseq2.recipes.config import GangSection, ModelSection\n",
```

Comment:
why defining it as 
```
dataset_config.name = "gsm8k_sft"
dataset_config.path = Path("/path/to/gsm8k_data/sft")
```
and not directly like
```
dataset_config = InstructionFinetuneDatasetSection(name = "gsm8k_sft", path = Path("/path/to/gsm8k_data/sft"))
```
?

Reply:
same question for `config = Config()  # instantiate an object`

Reply:
it would be interesting to say something about the expected data format in "/path/to/gsm8k_data/sft" !